packages/web/package.json

```diff
@@ -22,7 +22,8 @@
 		"prepublish": "yarn run build",
 		"test": "jest --watch",
 		"typescript": "tsc -p tsconfig.json",
-		"build-example": "scripts/build-example.sh 3.0.0-rc.0"
+		"build-example": "scripts/build-example.sh 3.0.0-rc.3",
```

Comment:
I think there should be some way to pick the version from version field of `package.json`.

Reply:
Yeah that will be great, any idea or lead how this can be done ? 
I searched but didnt got a good way to do this.

Reply:
Check this https://github.com/kentcdodds/nps
You can define the scripts outside of the package.json
Ref. https://github.com/appbaseio/appbase-js

Reply:
Fixed in #959